features/search_and_sort_users.feature

```diff
@@ -0,0 +1,32 @@
+Feature:
+
+  As an admin
+  I would like to search and sort users
+  to find a particular user amoung many
+  and to know which users are members and which are not
+
+  Background:
+    Given the following users exist
+      | first_name | last_name | email               | admin |
+      | John       | Adams     | ja@hotmail.com      | false |
+      | Sarah      | Conors    | sconors@example.com | false |
+      | Luke       | Skywalker | luke@force.net      | false |
+      | admin      | admin     | admin@shf.com       | true  |
+    And the following applications exist:
+      | user_email          | state                 | company_number |
+      | ja@hotmail.com      | waiting_for_applicant | 0000000000     |
+      | sconors@example.com | accepted              | 0000000000     |
+      | luke@force.net      | rejected              | 0000000000     |
+    And I am logged in as "admin@shf.com"
+
+  Scenario: Admin searches for luke
+    Given I am on the "users" page
+    When I fill in "q_email_cont" with "luke"
```

Comment:
This could be:
```
When I fill in "Profile Email" with "luke"
```

NOTE: We are trying to maintain out cucumber tests to be as close to describing actual user interaction as possible.  Thus we want to use actual field labels that appear on the screen, for instance, instead of HTML attributes that are not visible to the user.  

A user should be able to read a scenario and understand exactly what is going on, and confirm that the test represents - with fidelity - what they see on the screen.  (we are not using the tests for user acceptance at this point, but we still want to stick with the intent of cucumber in this regard).

Reply:
Keeping with your objective, I am still thinking about how to do this well. I wouldn't expect a user to understand code like `t("users.search_form.profile_email")`. The way I did it is not that good either. Could the ids be named better?

For ids:
If there are ever two things on the screen with the same name, the translations method will break; using an id ensures this will never happen. Ids are flexible enough to make them understood as to what they refer.

For translations:
Our Product 0wner(s) are already doing translations and may better understand the inline code.